libraries/AP_VisualOdom/AP_VisualOdom.cpp

```diff
@@ -91,16 +91,16 @@ const AP_Param::GroupInfo AP_VisualOdom::var_info[] = {
     AP_GROUPINFO("_VEL_M_NSE", 5, AP_VisualOdom, _vel_noise, 0.1),
 
     // @Param: _POS_M_NSE
-    // @DisplayName: Visual odometry position measurement noise
-    // @Description: Visual odometry position measurement noise in meters
+    // @DisplayName: Visual odometry position measurement noise 
+    // @Description: Visual odometry position measurement noise minimum (meters). This value will be used if the sensor provides a lower noise value (or not noise value)
```

Comment:
sorry to be picky, it's "no noise value" instead of "no**t** noise value"

Reply:
.. and this was my mistake when I made the original suggestion, sorry!

Reply:
It is OK. I did not notice it because English isn’t my first language